src/openforms/formio/api/serializers.py

```diff
@@ -3,12 +3,12 @@
 from rest_framework import serializers
 from rest_framework.reverse import reverse
 
-from openforms.submissions.models import TemporaryFileUpload
+from openforms.submissions.models import Submission, TemporaryFileUpload
 
 from .validators import MimeTypeValidator, NoVirusValidator
 
 
-class TemporaryFileUploadSerializer(serializers.Serializer):
+class TemporaryFileUploadSerializer(serializers.ModelSerializer):
```

Comment:
I don't see the benefit of changing this into a `ModelSerializer` if the record is created directly in the view anway.

What breaks if you don't do this?

Reply:
It was expected to be a `ModelSerializer` because fields were already defined in `Meta`, but I'll switch to `Serializer` and remove the irrelevant meta fields instead.